### PR TITLE
[5646] UI error when inputting data with keyboard inside Audit Logs -> Timestamp Filter

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5646

### CODE REVIEW

File: ui/app/src/components/AuditGridFilterPopover/AuditGridFilterPopoverBody.tsx [hunk 1]
```diff
@@ -179,15 +179,19 @@ export function AuditGridFilterPopoverBody(props: AuditGridFilterPopoverProps) {
                 inputFormat="MM/dd/yyyy"
                 value={fromDate}
                 onChange={onFromDateSelected}
-                renderInput={(props) => <TextField {...props} />}
+                renderInput={(props) => (
+                  <TextField {...props} inputProps={{ ...props.inputProps, onKeyDown: (e) => e.preventDefault() }} />
+                )}
               />
               <TimePicker
```
Comment (jvega190) [inline ui/app/src/components/AuditGridFilterPopover/AuditGridFilterPopoverBody.tsx]: @rart shouldn't this be a datePicker too?

**Review by rart — CHANGES_REQUESTED** [review]
Please apply changes as discussed during our call.

File: ui/app/src/components/AuditGridFilterPopover/styles.ts [hunk 2]
```diff
@@ -18,26 +18,18 @@ import { makeStyles } from 'tss-react/mui';
...
-    top: '-20px',
-    right: '-20px',
-    cursor: 'pointer',
-    '&:hover': {
-      opacity: '0.9'
+  timestampFiltersContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    marginBottom: theme.spacing(2),
+    '& .MuiTextField-root:first-child': {
```
Comment (jvega190) [inline ui/app/src/components/AuditGridFilterPopover/styles.ts]: There seems to be an issue in the `DateTime/date/time` pickers and the className doesn't apply (neither under PaperProps, or classes)

Comment (rart) [reply ui/app/src/components/AuditGridFilterPopover/styles.ts]: Use the field classes export

Comment (rart) [reply ui/app/src/components/AuditGridFilterPopover/styles.ts]: https://mui.com/x/react-date-pickers/getting-started/#MaterialUIPickers.js

**Review by rart — APPROVED** [review]

**Pull request merged**